app/node/node.py

```diff
@@ -0,0 +1,60 @@
+import os
+import sys
+import dotenv
+sys.path.insert(1,'../')
+from apache import proxyPass
+from apache import apache,apache_ssl
+from db import db_session
+from db.setup import *
+
+class node:
+    db_name=""
+    db_username=""
+    db_password=""
+    db_host="localhost"
+
+    def __init__(self,db_name,db_username,db_password,db_host):
+        self.db_name=db_name
+        self.db_username=db_username
+        self.db_password=db_password
+        self.db_host=db_host
+
+    def serve(self):
+        os.system("npm init")
+        os.system("touch .env")
+        content='''
+        DB_NAME={0}
+        DB_USERNAME={1}
+        DB_PASSWORD={2}
+        DB_HOST={3}
+        '''
+        f=open(".env","w")
+        f.write(content.format(self.db_name,self.db_username,self.db_password,self.db_host))
+        f.close()
+        os.system("npm install apache --save")
+        os.system("mv server_example.js server.js")
+        os.system("node server.js")
+
+    def host(self,ssh,confname,port,serverName,serverAlias,documentRoot,route,urlToMap):
+        if(ssh==False):
+            ape=apache.Apache()
+            ape.create_vhost(confname,port,serverName,serverAlias,documentRoot)
+            prox=proxyPass.ProxyPass(ape)
```

Comment:
Why are you passing apache object to proxyPass object ??. The constructor for the proxyPass class requires 0 object as argument da. That's why you got the issue.